test/blackbox-tests/test-cases/dune-init.t/run.t

```diff
@@ -144,6 +144,16 @@ Clean up from the dune file created at an absolute path
 
   $ rm -rf $PWD/_test_dir
 
+Add a library in the current working directory
+
+  $ dune init lib test_cwd_lib .
```

Comment:
Should these be in code block? If so, I can do that.

Reply:
This is the correct syntax for our cram-style tests. Thank you tho :)